zebra-network/src/peer_set/set.rs

```diff
@@ -996,11 +1001,17 @@ where
     /// Given a number of ready peers calculate to how many of them Zebra will
     /// actually send the request to. Return this number.
     pub(crate) fn number_of_peers_to_broadcast(&self) -> usize {
-        // We are currently sending broadcast messages to a third of the total peers.
-        const PEER_FRACTION_TO_BROADCAST: usize = 3;
+        if self.network.is_regtest() {
```

Comment:
Optional: It would be nice to always broadcast to all peers when a block that has been mined and submitted via the `submitblock` method is being advertised.

Reply:
Let;s leave this one out of this PR as it requires more work.